src/AisParser/AisParser.csproj

```diff
@@ -5,7 +5,7 @@
     <Copyright>Copyright Chris Richards</Copyright>
     <VersionPrefix>1.0.0</VersionPrefix>
     <Authors>Chris Richards</Authors>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Can this be changed back to netstandard2.0?

Reply:
FWIW, I'd probably favour netstandard too - if it doesn't need any particular Framework/Core dependencies then Standard [API] keeps it most flexible? (Albeit myself being no great expert on the wealth of MS SDK terminology n concepts...)

Reply:
The TargetFramework can be either netstandard or netcoreapp, it compiles as is with either target. The painful issue is that VS doesn't allow you to change it easily, you have to edit the file and restart VS.

Reply:
Darren, As far as I understand these things (which isn't necessarily a lot), netstandard is just an _API_ definition which - in many/most cases - framework or core sdk's should all support? If Chris doesn't insert anything not supported by one of those SDKs, then isn't NetStandard a decent/sensible 'target' for this library?
I'm unclear why you need to re-target it at NetCore. As it is, does it prevent usage in either a FrameWork or Core app? i.e. "VS doesn't allow you to change it easily" - not sure why you need to. (If I'm missing some pertinent point, then no offence intended)

Reply:
Could just be my quick referencing of a library in a new project, but VS complained about building a NetCore application with AisParser as a NetStandard library, IE, wouldn't compile...

You can set it to either framework, I'm good with either.

Cheers